### PR TITLE
[PICA-34] - Upgrade to polkadot v0.9.30

### PR DESCRIPTION
**Pull request by hussein-aitlahcen**
#### Intro

Upgrade polkadot from v0.9.27 to v0.9.30 as a checkpoint, then to v0.9.33 (latest version without workspace dependencies)

Require https://github.com/ComposableFi/centauri/pull/176 to be merged first.

#### Migrations
- v0.9.28
    - [x] https://github.com/paritytech/polkadot/pull/5582
        - Nomination not present in our runtimes
- v0.9.29
    - [x] https://github.com/paritytech/substrate/pull/12095
        - Nomination not present in our runtimes
- v0.9.30
    - [x] https://github.com/paritytech/substrate/pull/12034
        - BagList/Staking not present in our runtimes
    - [x] https://github.com/paritytech/polkadot/pull/5930
        - Nomination/BagList/Staking not present in our runtimes
    - [x] https://github.com/paritytech/substrate/pull/12230
        - Staking not present in our runtimes
    - [x] https://github.com/paritytech/polkadot/pull/5996
        - Staking not present in our runtimes
    - [x] https://github.com/paritytech/substrate/pull/12083
        - Contracts not present in our runtimes

### CODE REVIEW

**Review by vimukthi-git — APPROVED** [review]
LGTM :rocket:

**Review by obsessed-cake — APPROVED** [review]

File: code/parachain/frame/assets-registry/src/weights.rs [hunk 1]
```diff
@@ -14,28 +14,28 @@ pub trait WeightInfo {
 
 impl WeightInfo for () {
 	fn register_asset() -> Weight {
-		0
+		Weight::from_ref_time(10_0000)
```
Comment (cor) [inline code/parachain/frame/assets-registry/src/weights.rs:17]:
```suggestion
		Weight::from_ref_time(100_000)
```

File: code/parachain/frame/assets-registry/src/weights.rs [hunk 2]
```diff
@@ -14,28 +14,28 @@ pub trait WeightInfo {
...
 	}
 
 	fn update_asset() -> Weight {
-		0
+		Weight::from_ref_time(10_0000)
```
Comment (cor) [inline code/parachain/frame/assets-registry/src/weights.rs:21]:
```suggestion
		Weight::from_ref_time(100_000)
```

File: code/parachain/frame/assets-registry/src/weights.rs [hunk 3]
```diff
@@ -14,28 +14,28 @@ pub trait WeightInfo {
...
 	}
 
 	fn set_min_fee() -> Weight {
-		0
+		Weight::from_ref_time(10_0000)
```
Comment (cor) [inline code/parachain/frame/assets-registry/src/weights.rs:25]:
```suggestion
		Weight::from_ref_time(100_000)
```

**Review by cor — CHANGES_REQUESTED** [review]
LGTM! Just one nit, please fix the literals as per my suggestions

**Review by cor — APPROVED** [review]

**Review by benluelo — APPROVED** [review]
:rocket:

**Review by KaiserKarel — APPROVED** [review]

**Review by aeryz — APPROVED** [review]

**Review by Dom-Roth — APPROVED** [review]

**Review by Martin-Kiss — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Pull request merged**